### PR TITLE
Improve newsbot links

### PR DESCRIPTION
**Pull request by gundalow**
Add a few links so people have more context.

Question: What else should be added?

### CODE REVIEW

File: bots/newsbot-config.toml [hunk 1]
```diff
@@ -110,7 +113,7 @@ usual_reporters = []
 emoji = '⛏️' # :pick:
 name = 'devtools'
 title = 'DevTools'
-description = 'VScode extension, language server, linter, molecule, runner, navigator and potentially other development goodies'
+description = 'Projects to make it easier to write and test Ansible Content. Includes [VScode extension](https://github.com/ansible/vscode-ansible), [language server](https://github.com/ansible/ansible-language-server), [ansible-lint](https://github.com/ansible-community/ansible-lint), [molecule](https://github.com/ansible-community/molecule), [ansible-navigator](https://github.com/ansible/ansible-navigator) and potentially other development goodies. To see what's planned, and how you can help checkout the [foundation-devtools project board](https://github.com/orgs/ansible/projects/86/views/4)'
```
Comment (ssbarnea) [inline bots/newsbot-config.toml:116]: On that line there is a marker at the end of the line that looks like an error but I don't see why it appears as it looks valid to me.

**Review by ssbarnea — APPROVED** [review]

File: bots/newsbot-config.toml (hunk 1 above)
Comment (gundalow) [reply bots/newsbot-config.toml:116]: Oh, good spot. I think I need to change all the descriptions to double quotes, as `what's` is treated as end of string.

File: bots/newsbot-config.toml [hunk 2]
```diff
@@ -68,8 +71,8 @@ usual_reporters = []
 [[sections]]
 emoji = '📅' # :date:
 name = 'events'
-title = 'COMMUNITY EVENTS/MEETUPS'
-order = 107
+title = 'COMMUNITY EVENTS AND MEETUPS'
+order = 105
```
Comment (cybette) [inline bots/newsbot-config.toml:75]: Did you mean to move the order up? There are now two sections with order 105

**Comment by cybette** [comment]
Maybe update README.md with the same project description changes made in newsbot-config.toml?

File: bots/newsbot-config.toml (hunk 2 above)
Comment (gundalow) [reply bots/newsbot-config.toml:75]: woops, not, must have messed up when I rebased. Bad me for not looking at the diff.

**Review by cybette — APPROVED** [review]

File: bots/newsbot-config.toml [hunk 3]
```diff
@@ -92,7 +95,7 @@ usual_reporters = []
 emoji = '⚡️' # :zap:
 name = 'core'
 title = 'Ansible-Core'
-description = '**ansible-core** is a minimal package containing the base engine, modules, and plugins'
+description = "The `ansible-core` package contains the base engine, and a small subset of modules, and plugins. To see what's planned for the next release, look at the [`ansible-core` roadmaps](https://docs.ansible.com/ansible-core/devel/roadmap/ansible_core_roadmap_index.html)."
```
Comment (samccann) [inline bots/newsbot-config.toml:98]:
```suggestion
description = "The `ansible-core` package contains the base engine and a small subset of modules and plugins. To see what's planned for the next release, look at the [`ansible-core` roadmaps](https://docs.ansible.com/ansible-core/devel/roadmap/ansible_core_roadmap_index.html)."
```

File: bots/newsbot-config.toml [hunk 4]
```diff
@@ -101,7 +104,7 @@ usual_reporters = []
 emoji = '📦️' # :package:
 name = 'ansible'
 title = 'Ansible'
-description = '**ansible** is a batteries-included package providing a curated set of Ansible collections in addition to ansible-core'
+description = "The `ansible` package is a batteries-included package providing a curated set of Ansible collections, which includes `ansible-core`. To see when future releases are planned look at the [`ansible` roadmaps](https://docs.ansible.com/ansible-core/devel/roadmap/ansible_core_roadmap_index.html)."
```
Comment (samccann) [inline bots/newsbot-config.toml:107]:
```suggestion
description = "The `Ansible` package includes `ansible-core` and is a batteries-included package that provides a curated set of Ansible collections. See the [Ansible roadmaps](https://docs.ansible.com/ansible/devel/roadmap/ansible_roadmap_index.html) for future release plans."
```

File: bots/newsbot-config.toml [hunk 5]
```diff
@@ -119,7 +122,7 @@ usual_reporters = []
 emoji = '🐂' # :ox:
 name = 'antsibull'
 title = 'Antsibull'
-description = 'Tooling for building various things related to ansible'
+description = "Tooling for building the `ansible` package and collection documentation."
```
Comment (samccann) [inline bots/newsbot-config.toml:125]:
```suggestion
description = "Tooling for building the `Ansible` package and collection documentation."
```

File: bots/newsbot-config.toml (hunk 4 above)
Comment (gundalow) [reply bots/newsbot-config.toml:107]: If we are talking about the package should it be `Ansible` or `ansible`?

File: bots/newsbot-config.toml (hunk 5 above)
Comment (gundalow) [reply bots/newsbot-config.toml:125]: If we are talking about the package should it be `Ansible` or `ansible`?

File: bots/newsbot-config.toml (hunk 4 above)
Comment (samccann) [reply bots/newsbot-config.toml:107]: Docs has always referred to it as the Ansible package or Ansible. But I could go either way if `ansible` here is referring to the actual package name 'pip install ansible' for example. That said, I think maybe @dmsimard mentioned some distros install Ansible the full batteries package as `ansible` and others are only including core as `ansible`??

**Comment by gundalow** [comment]
Merged and live. Thank you for the reviews @cybette, @samccann and @ssbarnea

**Pull request merged**